index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="style.css">
+  <title>Livingi's Portfolio</title>
+</head>
+
+<body>
+  <header class="nav">
+    <nav class="nav-bar">
+
+      <div class="logo">
+        <a href="#">My logo</a>
+      </div>
+      <div class="menu-icon"><img src="images/bar.png" alt="menu navbar"></div>
+    </nav>
+  </header>
+    <section class="first-section">
+      <div class="port-graph">
+        <h1 class="head">
+          I'm Misheck, Glad to see you!
+        </h1>
+        <p class="paragraph">I’m a software developer! I can help you build a product , feature or website Look through
+          some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to
+          contact me.</p>
+
+        <p class="connect"> LET'S CONNECT</p>
+
+        <ul class="social-icons" >
+          <li><a href="#"><img src="./images/twitter.png" alt="social"> </a></li>
+          <li><a href="#"><img src="./images/Linkedin.png" alt="social"></a></li>
+          <li><a href="#"><img src="./images/medium.png" alt="social"></a></li>
+          <li><a href="#"><img src="./images/github.png" alt="social"></a></li>
+          <li><a href="#"><img src="./images/angellist.png" alt="social"></a></li>
+        </ul>
```

Comment:
- [ ] Kindly, stick to the design as much as possible for better UI (user interface) and this is a good professional practice that every frontend software developer should maintain. Please, make sure to make social media icons closely similar to the [original design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).

**Current mobile menu**           |  **Original design**
:-------------------------:|:-------------------------:
![](https://user-images.githubusercontent.com/80895497/153435947-dccf4470-9e34-418b-b6ff-b53c4b6fcb78.PNG)  |  ![](https://user-images.githubusercontent.com/80895497/153435943-31421031-2545-49a0-934b-d6f9a5951bb7.PNG)